_about/release-cadence.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Build & Release Cadence
+description: How we manage, number, and support Istio releases.
+weight: 5
+---
+
+We produce new builds of Istio on a daily basis.
+Once a week, one of these daily build undergoes additional
+qualification and gets anointed the weekly release. Once a month, one of the weekly releases undergo
+additional qualifications and gets anointed as the monthly release. Once a quarter or so, one of the monthly
+releases undergoes additional qualifications and gets anointed as a long term supported (LTS) release.
+
+Each release type implies different levels of quality and support:
+
+|Release Type | Support Level | Typical Use
+|-------------+---------------+------------
+|Daily   | No support           | Dangerous, may not be fully reliable. Useful to experiment with.
+|Weekly  | No support           | Dangerous, may not be fully reliable. Useful to experiment with.
+|Monthly | Support is provided for the current monthly release only | Safe to deploy in production. Usually only adopted by bleeding edge users, or users seeking specific features.
```

Comment:
Do we really want to say monthly is safe to deploy in prod?

Reply:
That came from this doc: https://docs.google.com/document/d/1scC_f7nObwBut_xxE6lUfvcHBybmttUsJDMwXD3gcO4/edit#heading=h.ob68p14f45rs

Let's talk about it during the TOC meeting.